orm/versioning_bucket.go

```diff
@@ -2,11 +2,14 @@ package orm
 
 import (
 	"bytes"
+	"encoding/binary"
 
 	"github.com/iov-one/weave"
 	"github.com/iov-one/weave/errors"
 )
 
+const versionMaxBytes = 4
```

Comment:
I think this is not max bytes, but exact size in bytes of the serialized version number.

Reply:
Indeed, iterating on that constant's name made it worse.